config/environments/production.rb

```diff
@@ -13,7 +13,8 @@
 
   # Disable serving static files from the `/public` folder by default since
   # Apache or NGINX already handles this.
-  config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
+  require "miq_environment"
+  config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present? || !MiqEnvironment::Command.is_production_build?
```

Comment:
I kept the original ENV var check as it seems to be a vanilla Rails thing, and the feature is reasonable to keep.

Reply:
If we do this, it would be difficult to run in production mode with apache not on a pod/appliance/container .  I vaguely recall some people doing this in the past.

It feels like we should honor the value for `ENV['RAILS_SERVE_STATIC_FILES']`.

Maybe `RAILS_SERVE_STATIC_FILES=false bin/rails s` should honor it?

Reply:
I'm not sure I follow.  This and the last comment make me think your logic is backwards?

Reply:
How would you disable this in a non-production build? It's a weird use case I believe the container people used at one point where they ran a dev setup with apache on their laptops and sometimes they'd run in production mode.

I was a bit surprised to see the ENV variable's possible `false` value ignored.